libs/application/templates/financial-aid/src/fields/MissingFiles/MissingFiles.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { useEffect, useMemo, useState } from 'react'
+import { useIntl } from 'react-intl'
+
+import { Text, Box, AlertMessage, Input } from '@island.is/island-ui/core'
+import {
+  ApplicationEventType,
+  ApplicationState,
+  FileType,
+  getCommentFromLatestEvent,
+} from '@island.is/financial-aid/shared/lib'
+import { getValueViaPath } from '@island.is/application/core'
+
+import { filesText, missingFiles } from '../../lib/messages'
+import { Files } from '..'
+import { FAFieldBaseProps, UploadFileType } from '../../lib/types'
+import useApplication from '../../lib/hooks/useApplication'
+import { Controller, useFormContext } from 'react-hook-form'
+import { useFileUpload } from '../../lib/hooks/useFileUpload'
+import DescriptionText from '../DescriptionText/DescriptionText'
+
+const MissingFiles = ({
+  application,
+  setBeforeSubmitCallback,
+  field,
+}: FAFieldBaseProps) => {
+  const { currentApplication, updateApplication } = useApplication(
+    application.externalData.veita.data.currentApplicationId,
+  )
+  const isSpouse = getValueViaPath(field as any, 'props.isSpouse')
```

Comment:
I turned this into `field as RecordObject<any>`. I couldn't get away with skipping the any just because of how the field object is structured and the arguement type of getValueViaPath